roles/sft-server/defaults/main.yml

```diff
@@ -80,3 +80,9 @@ systemd_coredump_max_file_size: '4G'
 
 # diffie-hellman keysize for nginx
 sft_nginx_dh_keysize: 2048
+
+# NOTE: disabling rate limit might be desired for on-prem scenarios
+sft_nginx_rate_limit_disabled: false
```

Comment:
nitpick: I prefer positively-phrased boolean variables, i.e. `sft_nginx_rate_limit_enabled: true` and `{% if (sft_nginx_rate_limit_enabled | bool) -%}`. Double negation is harder to mentally parse, I find.

Reply:
This is indeed kinda ridiculous and it's even more stupid that I fail to remember why I did it that way. *Adjusted*